topics/wallet-iteration-standard/ledger-extension.md

```diff
@@ -0,0 +1,91 @@
+# ICRC-27: Ledger extension for Wallet Interaction Standard
+
+| Status |
+|:------:|
+| Draft  |
+
+## Summary
+
+This specification describes a ledger extension for the ICRC-25 Wallet Interaction Standard.
+
+## Terminology
+
+* wallet: A service that manages a user's keys and can sign and perform canister calls on their behalf.
+* relying party: A service that wants to request calls on a specific canister.
+
+## Messages
+
+### `permission`
+
+Additional `ledger` scope is passed in the request and in the response an additional `subaccounts` property is returned in each `identity` when the wallet supports this extension.
```

Comment:
I think `icrc1_ledger` would be better to make it clear where the subaccount concept is defined.

Reply:
Yeah in that case we can drop the `standards` list from each subaccount, making this spec limited to the icrc-1 subaccounts. Which gives more flexibility for other subaccount standards with completely different concepts to define their own scope and spec.